applications/dashboard/src/scripts/compatibilityStyles/groupsStyles.ts

```diff
@@ -106,6 +107,17 @@ export const groupsCSS = () => {
         },
     });
 
+    cssOut(`.Group-Header.NoBanner .Photo.PhotoWrap.PhotoWrapLarge.Group-Icon-Big-Wrap`, {
```

Comment:
Are all of these extra classes needed? It feels a bit overspecific.

Maybe something like this would work?

```suggestion
    cssOut(`.Group-Header.NoBanner .Group-Icon-Big-Wrap`, {
```